src/Support/Contracts/CacheKeyAndTags.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Contracts;
+
+use Illuminate\Contracts\Auth\Authenticatable;
+
+interface CacheKeyAndTags
+{
+    /**
+     * Generate the cache key.
+     *
+     * @param int|string|null      $id
+     * @param array<string, mixed> $args
```

Comment:
```suggestion
     * @param  int|string|null $id
     * @param  array<string, mixed> $args
```

Please do the same for the other methods